app/src/main/java/com/github/libretube/db/dao/SubscriptionGroupsDao.kt

```diff
@@ -11,6 +11,9 @@ interface SubscriptionGroupsDao {
     @Query("SELECT * FROM subscriptionGroups")
     suspend fun getAll(): List<SubscriptionGroup>
 
+    @Query("SELECT * FROM subscriptionGroups WHERE name = :name")
+    suspend fun get(name: String): SubscriptionGroup?
```

Comment:
```suggestion
    @Query("SELECT EXISTS(SELECT * FROM subscriptionGroups WHERE name = :name)")
    suspend fun exists(name: String): Boolean
```

Reply:
Interesting, didn't know this existed (pun intended), thanks!